ptf-tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ptf
+paramiko
+scapy==2.5.0

```

Comment:
The scapy version present in ptf-tests repo is 2.4.5

Hopefull this doesnt break anything with a different version used

Reply:
Any thoughts on this @ffoulkes

Reply:
I generally take Dependabot's recommendations, on the (perhaps foolish) assumption that it knows what it's doing.

Scapy 2.4.5 was released Apr 19, 2021. [Scapy 2.5.0](https://github.com/secdev/scapy/releases) was released Dec 25, 2022. The latter adds support for Python 3.9 and 3.10 (which might be important) and has some TLS fixes. It also has a breaking change to `FlagField` (which might be an argument against).

There's one entry in the GitHub Advisory Database, for CVE-2019-1010262. It was fixed in v2.4.2.

I'd suggest running the question by the test team.

Reply:
@singhal-kamalkant can you provide some feedback if these versions are ok to upgrade to from ptf-tests perspective

Reply:
@5abeel submodule update is failing in jenkins with this
hudson.plugins.git.GitException: Command "git config --get submodule.ptf-tests.url" returned status code 1